### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hfcjweinstock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # EncompassRest
 Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
 
-A MAJOR Redesign has been pushed out. The old source code is still available in the [legacy branch](https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation and a NuGet package forthcoming.
+A MAJOR Redesign has been pushed out. The old source code is still available in the [legacy branch](https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation forthcoming.
+
+NUGET:[Package](https://www.nuget.org/packages/EncompassRest)
 
 ## Getting Started
 1. Download the source code from this repository and add the `EncompassRest` project file to your Visual Studio 2017 solution. This will be much simpler when the NuGet package gets published and will support older Visual Studio versions.
```
Comment (TylerBrinkley) [inline README.md]: Could you update this line to simply 1. Install the `EncompassRest` NuGet package.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # EncompassRest
 Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
 
-A MAJOR Redesign has been pushed out. The old source code is still available in the [legacy branch](https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation and a NuGet package forthcoming.
+A MAJOR Redesign has been pushed out. The old source code is still available in the [legacy branch](https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation forthcoming.
```
Comment (TylerBrinkley) [inline README.md]: Can we remove the redesign mention? Seems unnecessary now.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # EncompassRest
-Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
+Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+ and 2.0.
 
-A MAJOR Redesign has been pushed out. The old source code is still available in the [legacy branch](https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation and a NuGet package forthcoming.
+(https://github.com/EncompassRest/EncompassRest/tree/legacy). More documentation forthcoming.
```
Comment (TylerBrinkley) [inline README.md]: Remove legacy link

File: README.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
+
+NUGET:[Package](https://www.nuget.org/packages/EncompassRest)
```
Comment (TylerBrinkley) [inline README.md]: Is this necessary with the link below?

File: README.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 # EncompassRest
-Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
+Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+ and 2.0.
```
Comment (TylerBrinkley) [inline README.md]: We should only display the minimums, thus the plus signs. The fact that there's now a 2.0 specific build doesn't need to be expressed here, they'll see it on NuGet.

File: README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 # EncompassRest
-Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
+Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+
```
Comment (TylerBrinkley) [inline README.md]: I know this is petty but we lost a period here.

Comment (hfcjweinstock) [reply README.md]: hmm.  i thought it looked silly there. ill put it back.

**Pull request merged**